examples/multigpu/node_classification_sage.py

```diff
@@ -145,6 +146,8 @@ def layerwise_infer(
 ):
     model.eval()
     with torch.no_grad():
+        if not use_uva:
+            g = g.to(device)
```

Comment:
It means in our benchmark, entire graph wil be put into GPU?

Reply:
Yes. It's because dgl uses inference to do the final testing so this part won't be considered in the comparison. We can optimize this part by putting the graph into GPU. It's also the original way.